README.md

```diff
@@ -1 +1,22 @@
-# hw01
\ No newline at end of file
+##### HOMEWORK ASSIGNMENT 1
+I'm **Jiayang He**, a MAPSS student concentrating on *psychology*.
+
+My interest in the area focuses on several specific topics under certain sub-fields of psychology:
+
+* Psychophysiology
+  + Will stress be contagious via watching stressful videos? 
+* Social neuronscience
+  + How social stress impact interoception?
+* Clinical psychology
+  + Will people with depresssion or panic disorder have a different pattern of stress response?
+
+I am currently working in Dr. Norman's SPNL (Social Psychophysiology & Neuroendocrinology Lab). Here's our website: <https://sites.google.com/a/uchicago.edu/spnl/>.
+
+Besides schoolwork, recently I have had a lot of fun playing a game called *__Animal Crossing__*, and I would totally recommend this game to you. 
+![Image of Animal Crossing](https://s1.gaming-cdn.com/images/products/4809/orig/animal-crossing-new-horizons-switch-cover.jpg)
+
+
+For this homework assignment, I first forked the repository from the link provided on the homework web page to my own github. Then I cloned the link and used it to create a new project in RStudio. I opened the README file in RStudio and made edits to it following the instructions on R Markdown Cheatsheet. After making changes to the file, I clicked to see the difference, staged README to commit it, and pushed it to Github. Then I created a new pull request. 
+
+To be honest, I find this experience of the Github workflow is somewhat difficult since this is my very first exprience using Github. It is kind of tricky if there's no prior understanding of how this works. However, the tutorial video and TA's office hours are so helpful! I would thank Deb for her help on understanding Markdown syntax. To me, Markdown is similar to Word, except it asks the user to use codes rather than click on the buttons to change formats of the writing.
```

Comment:
Good feedback. In the future, put this in the pull request comment rather than the `README.md`